app.go

```diff
@@ -85,3 +88,11 @@ func Automatic(opts Options) *App {
 
 	return a
 }
+
+func (a *App) addDefaultRouteNames() {
```

Comment:
I still don't quite understand what this means, sorry. Also, the fact it happens on every request isn't great either.